guides/doc-Upgrading_and_Updating/topics/upgrading_external_database.adoc

```diff
@@ -0,0 +1,25 @@
+[id="Upgrading_External_Database_{context}"]
+= Upgrading External Database
+
+You can upgrade an external database from {EL} 7 to {EL} 8 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+
+.Prerequisites
+. Create a backup.
++
+* On a virtual machine, take a snapshot.
```

Comment:
Does this make sense? You're creating a backup anyway (to transfer to the new machine) and you're not making changes to the old host. So why is a snapshot needed?

In other words, move "Create a backup" from a Prerequisite to a step in the Procedure.